### PR TITLE
@microsoft packages and t2 migration

### PR DESCRIPTION
**Pull request by nturinski**
Pretty straight forward

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -307,18 +307,19 @@
...
+        "vscode-azureextensiondev": "^0.10.4",
         "vscode-test": "^1.5.2",
         "webpack": "^5.28.0",
         "webpack-cli": "^4.6.0"
     },
     "dependencies": {
-        "@azure/arm-resources": "^3.0.0",
+        "@azure/arm-resources": "5.0.0",
         "@azure/arm-resources-profile-2020-09-01-hybrid": "^1.0.0",
+        "@microsoft/vscode-azext-azureutils": "^0.1.0",
```
Comment (alexweininger) [inline package.json:318]: We should update this to the latest version `0.1.1`

File: package.json [hunk 2]
```diff
@@ -307,18 +307,19 @@
         "ts-node": "^7.0.1",
         "typescript": "^4.3.5",
         "vsce": "^1.87.0",
-        "vscode-azureextensiondev": "^0.10.0",
+        "vscode-azureextensiondev": "^0.10.4",
```
Comment (alexweininger) [inline package.json:310]: Probably worth using the `@microsoft/vscode-azext-dev` package I just published.

**Review by alexweininger — DISMISSED** [review]

**Review by wwlorey — DISMISSED** [review]

**Review by alexweininger — COMMENTED** [review]
Update package name here https://github.com/microsoft/vscode-azureresourcegroups/blob/8ad504cf26b662a9c206704b31bd993620d6240a/gulpfile.ts#L9

**Comment by nturinski** [comment]
I hate when I forget to save in VS Code >:(

**Review by alexweininger — APPROVED** [review]

**Pull request merged**